data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaCustomConsumerFactory.java

```diff
@@ -136,7 +136,6 @@ private Properties getConsumerProperties(final KafkaConsumerConfig sourceConfig,
         }
         setConsumerTopicProperties(properties, topicConfig, topicConfig.getGroupId());
         setSchemaRegistryProperties(sourceConfig, properties, topicConfig);
-        LOG.debug("Starting consumer with the properties : {}", properties);
```

Comment:
We can consider just having log without properties.

LOG.info("Starting kafka consumer");

Reply:
This is also a good thing to have.

Reply:
When is KafkaCustomConsumerFactory vs KafkaSource code path execute ?

Reply:
KafkaCustomConsumerFactory path is for Kafka sink. I think this should still be kept at debug level